pkgs/development/compilers/swift/wrapper/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, swift
+, wrapperParams ? swift._wrapperParams
+, useSwiftDriver ? true, swift-driver
+}:
+
+stdenv.mkDerivation (wrapperParams // {
```

Comment:
Good point. I think the reason it is an arg at all is because I was concerned about someone trying to use the wrapper with an overridden `swift` arg, where `swift._wrapperParams` may not be present and needs to be provided otherwise. Feels hacky, though.

It can be flipped if we handle `swiftDriver`, which is the only thing that depends on order. So we could maybe do:

```nix
stdenv.mkDerivation ({ ... } // wrapperParams // { swiftDriver = ... })
```

Though, thinking about it some more, if the intention is to use `wrapperParams` to override something in the derivation, I think there is no benefit over just using `overrideAttrs`?

Reply:
If we can drop it, let's drop it. Otherwise, I'm fine with the order.

Reply:
The arg was dropped.